index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jim Ntare | Software Engineer</title>
+    <link rel="stylesheet" href="./build/styles.css" />
+  </head>
+  <body>
+    <header class="nav_bar">
+      <div class="logo">Jim Ntare.</div>
+      <div class="nav_items">
+        <div class="nav_item">Portfolio</div>
+        <div class="nav_item">About</div>
+        <div class="nav_item">Contact</div>
+      </div>
+    </header>
+    <h1>I'm Jim</h1>
+    <h1>Glad to see you!</h1>
+    <p>
+      I'm a Software Developer! I can help you build a product, feature or
+      website. Look through some of my work and experience! if you like what you
+      see and have a project you need coded, don't hesitate to contact me.
+    </p>
+    <p>let's connect</p>
```

Comment:
- [ ] Here you have to follow the design from the Figma. You are asked to create the headline section (right after the header):

This is the given design
![figma](https://user-images.githubusercontent.com/53124548/133518060-4955a9e0-b794-4b1c-8f5b-41825d2a99bf.png)


Your project

![yours](https://user-images.githubusercontent.com/53124548/133518160-a84cf889-e074-416f-8017-8735e3ad83a7.png)

- [ ] Do not forget the background image for this section (Try as much as possible to go through the project requirements, doing so can save you a lot of time)